login/templates/registration/register.html

```diff
@@ -2,14 +2,19 @@
 
 <!--Register-->
 <div class="container py-5">
-    <h1>Register</h1>
-    <form method="POST">
-        {% csrf_token %} {{ register_form|crispy }}
-        <button class="btn btn-primary" type="submit">Register</button>
-    </form>
-    <p class="text-center">
-        If you already have an account, <a href="/login">login</a> instead.
-    </p>
+  <h1>Register</h1>
+  <form method="POST">
+    {% csrf_token %} {{ register_form|crispy }}
+    <p>Please select your Roll:</p>
```

Comment:
Role*

Also why use a form here and not simple links? The complications this adds to the view code seem unnecessary

Reply:
yes, you are right, this page is no longer in use because the registration got splite between the student and recruiter. I will upload a PR without this page and URL link.

Reply:
done